### PR TITLE
fb url added!

### PR DESCRIPTION
**Pull request by raihan-uddin**

### CODE REVIEW

File: startbootstrap-freelancer-gh-pages/index.html [hunk 1]
```diff
@@ -244,26 +244,23 @@ <h1 class="name">Start Bootstrap</h1>
...
                             <li>
-                                <a href="#" class="btn-social btn-outline"><span class="sr-only">Twitter</span><i class="fa fa-fw fa-twitter"></i></a>
+                                <a href="https://www.linkedin.com/in/saifulazad" class="btn-social btn-outline"><span class="sr-only">Linked In</span><i class="fa fa-fw fa-linkedin"></i></a>
                             </li>
                             <li>
-                                <a href=" linkedin.com/in/saifulazad" class="btn-social btn-outline"><span class="sr-only">Linked In</span><i class="fa fa-fw fa-linkedin"></i></a>
-                            </li>
-                            <li>
-                                <a href="#" class="btn-social btn-outline"><span class="sr-only">Dribble</span><i class="fa fa-fw fa-dribbble"></i></a>
+                                <a href="http://stackoverflow.com/users/1919324/saiful-azad" class="btn-social btn-outline"><span class="sr-only">Linked In</span><i class="fa fa-stack-overflow"></i></a>
```
Comment (saifulazad) [inline startbootstrap-freelancer-gh-pages/index.html]: Linked In ?

Comment (raihan-uddin) [reply startbootstrap-freelancer-gh-pages/index.html]: Yes
I have just added https://www. before linkedin.com

Comment (saifulazad) [reply startbootstrap-freelancer-gh-pages/index.html]: <span class="sr-only">Linked In</span>
I think i should b stackoverflow

Comment (raihan-uddin) [reply startbootstrap-freelancer-gh-pages/index.html]: Yah Right...
sorry for my mistake .
I have now fixed it.

**Pull request merged**